### DISCUSSION TITLE
16.1.1 - Blink Build Beta is out!

### DISCUSSION THREAD

**Discussion by carloscabanero** [blinksh/blink#1698] [Announcements]
-

**Comment by jaclu** [comment]
Will there be any documentation for Build up Python?
Currently only Next.js has any content.

**Comment by carloscabanero** [comment]
Yes, definitely. We wanted people to start using it and we will be filling out the gaps this week.

At the moment the Hacker Tools image already includes the latest Python 3 and 2. Are you interested on any frameworks or tooling in special?

**Reply by jaclu** [reply]
Django and Jupyter would probably make sense to use as sample frameworks

**Reply by jaclu** [reply]
It would be helpful to see a bit more details on what limits the Build host has, like CPUs & Ram, storage, and bandwidth limits.

In order to get a feeling about how complex data processing tasks can realistically be handled in the Build environment when working on new solutions and services. 

I am not mixing up production and dev environments, obviously, production would run elsewhere, but sometimes you need to do your work on huge datasets or require significant processing in order to see that it can handle larger tasks, and then this becomes a limiting factor if you can't (if need be) temporarily scale up a normally reasonably sized dev-env.

**Reply by carloscabanero** [reply]
Thanks! I will do both as I think both are very interesting. For Jupyter I actually found using it with VS Code a very good combo, but if you have any experience maybe you can point me on whether the regular Jupyter interface may be better for this scenario.

On the machine side of things, we are planning this to be the basic plan, more for casual use. The machine has 2vcpu and 4GB RAM. We could really go up to whatever we would feel like. I found for my personal usage that the basic paired with the possibility to scale to 16GB and 32GB when testing heavy projects is a good sweet spot.

**Reply by jaclu** [reply]
That sounds more than reasonable for normal cases, and if there is a possibility to scale up when needed it would really be a fully capable Dev-Env for the iPad!
